spotlight-client/src/charts/BubbleChart.tsx

```diff
@@ -72,11 +72,13 @@ const LegendWrapper = styled.div`
 type BubbleChartProps = {
   data: CategoricalChartRecord[];
   height: number;
+  preview?: boolean;
```

Comment:
This is a design rather than an implementation issue, but I don't actually think we should be hiding the legend on this preview. The chart is totally unreadable without it. We include the legend for other preview charts where needed, so there is precedent for this. I will appeal to @hobuobi for confirmation on this, though.

Failing that, could we call this prop "showLegend" or something instead? "preview" is not really meaningful in this context, the charts in this directory are meant to be as generic as possible.

Reply:
I tend to agree! No need to get rid of the legend.